service/status-report/main.go

```diff
@@ -207,7 +207,7 @@ func setClusterInformation(ctx context.Context, cl client.Client, status interfa
 
 func setStorageQuotaUtilizationRatio(ctx context.Context, cl client.Client, status interfaces.StorageClientStatus) {
 	clusterResourceQuota := &quotav1.ClusterResourceQuota{}
-	clusterResourceQuota.Name = status.GetClientName()
+	clusterResourceQuota.Name = fmt.Sprintf("storage-client-%s-resourceqouta", utils.GetMD5Hash(status.GetClientName()))
```

Comment:
please use helper function to get the Name so that we will have single source of truth to generate names, if not we are going to get into the problems where we are generating names in different places with  strings